lib/libc/sys/kqueue.2

```diff
@@ -250,6 +250,21 @@ Filters may set this flag to indicate filter-specific EOF condition.
 See
 .Sx RETURN VALUES
 below.
+.It Dv EV_KEEPUDATA
+Causes
+.Fn kevent
+to leave unchanged any 
+.Fa udata
+associated with an existing event.  This allows other aspects of the
+event to be modified without requiring the caller to know the
+.Fa udata
+value presently associated.
+This is especially useful with
+.Dv NOTE_TRIGGER
+or flags like
+.Dv EV_ENABLE.
+This flag is ignored for initial event registration with
```

Comment:
Or you could fail `EV_ADD | EV_KEEPUDATA` with `EINVAL`.  I probably have a slight preference for that vs ignoring it.

Reply:
OK, can do.